src/haz3lweb/LocalStorage.re

```diff
@@ -41,51 +41,51 @@ module Settings = {
   };
 };
 
-// LangDocMessages serialization
-module LangDocMessages = {
-  let save_langDocMessages_key: string = "LANGDOCMESSAGES";
+// ExplainThisModel serialization
+module ExplainThisModel = {
+  let save_ExplainThisModel_key: string = "ExplainThisModel";
 
-  let serialize = langDocMessages =>
-    LangDocMessages.serialize(langDocMessages);
+  let serialize = explainThisModel =>
+    explainThisModel |> ExplainThisModel.sexp_of_t |> Sexplib.Sexp.to_string;
 
   let deserialize = data =>
-    try(LangDocMessages.deserialize(data)) {
```

Comment:
This will mean that if a form is removed from ExplainThisForm that had saved data for it, the ExplanThisModel won't be loaded in.